### PR TITLE
fix: retrieve subgraph urls from sdk config file

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
retrieve subgraph (dev) urls from sdk instead of from frontend file

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from bal_tools.ecosystem import Aura
 from bal_tools.safe_tx_builder import SafeTxBuilder
 
-exempt_chains = ["fantom", "mode", "linea", "fraxtal"]
+exempt_chains = ["fantom", "mode", "linea", "fraxtal", "goerli"]
```
Comment (Tritium-VLK) [inline tests/conftest.py:10]: We actually have subgraphs on fraxtal and mode now if you want to keep going, it's not part of this PR really though.

**Review by Tritium-VLK — COMMENTED** [review]
Do we know what the resulting outputs will look like in the subgraph json in bal_addresses.  Are we sure we wont' leak keys?

File: tests/conftest.py (hunk 1 above)
Comment (gosuto-inzasheru) [reply tests/conftest.py:10]: https://github.com/BalancerMaxis/bal_tools/issues/19

**Comment by gosuto-inzasheru** [comment]
@Tritium-VLK no keys involved in this solution

**Review by Tritium-VLK — APPROVED** [review]
